interview-questions.md

```diff
@@ -14,44 +14,44 @@ Researched answer:
 
 1. What is git? What is the difference between git and Github?
 
-Your answer:
+Your answer: Git is how we use version control to share and collaborate creating code. Github is the repository that stores the code.
 
-Researched answer:
+Researched answer: Git is a version control tool used to save code to a local drive and uses commands such as push and pull, commit, reset, fetch, and merge to save changes. GitHub is system where repositories are created with unique urls and give the place where files are stored in a graphical interface.
 
 2. Which JavaScript operators will return a Boolean value?
 
-Your answer:
+Your answer: An operator that would have an output of true or false.
 
-Researched answer:
+Researched answer: Logical operators such as && (and) or || (or). When || (or) is used, if any part of the arguments are true then it returns a true output.  When && (and) is used, both arguments have to be true to return a true output.  If any argument is false then it will return a false output. 
 
-1. What is the difference between an array's index and its value?
+3. What is the difference between an array's index and its value?
 
-Your answer:
+Your answer: The index is the position the data is at in the array, defined by a number starting from zero. The value is the actual data that is listed in the array.
 
-Researched answer:
+Researched answer: An array's index is the ordered location of a value within the array.  The value is the data being listed in the array.
 
 4. Compare and contrast arrays and strings. Please include ways in which they are the same and ways in which they are different.
 
-Your answer:
+Your answer: Arrays are different data types grouped together. String is one data type. 
 
-Researched answer:
+Researched answer: Both are used to store data and could use an index.  String is a primative data type. Arrays use different data types that could include strings. Arrays can use mutators to change their values and strings cannot.
```

Comment:
Would be good to mention the differences in which they are written as well, brackets vs quotes.